.github/pull_request_template.md

```diff
@@ -1,51 +1,40 @@
-## What does this change?
-
 <!--
-**NOTE**: This is meant as a guide and memory jog, not a prescriptive list of tasks you must do!
+This is meant as a guide and memory jog, not a prescriptive list of tasks you must do!
 Feel free to modify as suits, just please do read over and make sure you've at least thought about the points and if there's any impact.
 -->
 
 <!--
-Add details of the change here. A PR should have enough detail to be understandable far in the future. e.g what is the problem/why is the change needed, how does it solve it and any questions or points of discussion.
+Append the ticket *task ID* to the PR description like so: Awesome update [GEN-123]
 
-Add any relevant tickets or links, eg: Resolves [ticket](http://link-to-ticket)
+Notion will detect this and add a link to the original ticket as a comment.
 -->
 
 ## Testing
 
-- [ ] I wrote new unit tests for this change
-- [ ] I wrote new integration/e2e tests for this change
-- [ ] I updated existing tests for this change
-- [ ] I ran test commands to test this change (list below)
-
-```
-npm run test
-```
+- [ ] I have tested this change against all known scenarios and considered new ones.
+  - E.g. net negative import scenario for solution summary graph.
+- [ ] I have tested this change against real data
 
 ## Monitoring
 
 - [ ] This change can be deployed to prod without triggering any alarms
 - [ ] This change will be covered by our existing monitoring
-      (no new /metrics/dashboards/alarms are required)
+  - No new CloudWatch alarm or Sentry logging is required
 - [ ] This change will have no (or positive) effect on resources and/or limits
-      (including CPU, memory, AWS resources, calls to other services)
-- [ ] This change requires the cache to be flushed after deployment
-  - Usually applicable for APIs when a DTO has been updated
+  - Including CPU, memory, AWS resources, calls to other services
 
 ## Rollout
 
-- [ ] This change can be merged immediately into the pipeline upon approval (no upstream/downstream/config updates required first)
+- [ ] This change can be merged immediately into the pipeline upon approval
+  - No upstream/downstream/config updates required first
 - [ ] All dependent changes are already deployed to prod
-- [ ] This change can be rolled back without any issues after deployment to prod
-
-## UI
-
-<!-- Usually only applicable to UI changes: what did it look like before and what will it look like after? -->
 
-### Design review
+## Rollback
 
-- [ ] I requested review from jacklsmith
```

Comment:
Removed this as I think its evident from the reviewers section of the PR if this has been done